packages/react/src/components/header/components/headerActionBarSubItem/HeaderActionBarSubItem.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { cloneElement, forwardRef, ReactNode } from 'react';
+import { VisuallyHidden } from '@react-aria/visually-hidden';
+
+import classes from './HeaderActionBarSubItem.module.scss';
+import classNames from '../../../../utils/classNames';
+import { IconLinkExternal } from '../../../../icons';
+import parentClasses from '../headerActionBarItem/HeaderActionBarItem.module.scss';
+
+export interface HeaderActionBarSubItemProps extends React.HTMLAttributes<HTMLButtonElement> {
+  /**
+   * Aria label for the item.
+   */
+  ariaLabel?: string;
+  /**
+   * Icon element (on the left side of the label) for the action bar item.
+   */
+  iconLeft?: ReactNode;
+  /**
+   * Icon element (on the right side of the label) for the action bar item.
+   */
+  iconRight?: ReactNode;
+  /**
+   * Count for the right aligned red notification circle
+   */
+  notificationCount?: number;
+  /**
+   * Aria label for notification number.
+   */
+  notificationCountAriaLabel?: string;
+  /**
+   * Label for the action bar item.
+   */
+  label?: string | JSX.Element;
+  /**
+   * If heading style is used
+   */
+  heading?: boolean;
```

Comment:
This is obsolete now after SubItemGroup was added?

If still needed, `heading` indicates it should be a string, "isHeading" would be clearer. If there is "label" and "heading", so user's might be confused.

Reply:
its needed, but I changed it to `isHeading`

Reply:
Hmm... I was thinking that no prop would be needed, just by ahndling this in `dropdownSubItem`'s 1st child would do the trick also without the need of a prop?